src/utils/assets/elements/touchable/coin.py

```diff
@@ -1,9 +1,20 @@
-from src.utils.assets.create_images import create_image
-from src.utils.constants import COIN_SIZE
+from pygame import image, transform
+
+from src.utils.constants import GENERAL_SIZE
 from src.utils.directories import COIN_DIR
 
+COIN_1 = transform.scale(
+    image.load(COIN_DIR + "coin_1.png"), GENERAL_SIZE
+)
+
+COIN_2 = transform.scale(
+    image.load(COIN_DIR + "coin_2.png"), GENERAL_SIZE
+)
+
+COIN_3 = transform.scale(
+    image.load(COIN_DIR + "coin_3.png"), GENERAL_SIZE
+)
 
-COIN_1 = create_image(COIN_DIR + "coin_1.png", COIN_SIZE)
-COIN_2 = create_image(COIN_DIR + "coin_2.png", COIN_SIZE)
-COIN_3 = create_image(COIN_DIR + "coin_3.png", COIN_SIZE)
-COIN_4 = create_image(COIN_DIR + "coin_4.png", COIN_SIZE)
+COIN_4 = transform.scale(
```

Comment:
You are making unnecessary changes, for example you are changing the size of the coins, since by giving them the size of 60 x 60 pixels, you will change their shape and it will be super ugly.

Reply:
The size of the object is centered inside each box of the screen where the size is 50x50 and by doing it that way the objects are not properly accommodated.